src/link/Coff/lld.zig

```diff
@@ -184,9 +184,10 @@ pub fn linkWithLLD(self: *Coff, arena: Allocator, prog_node: *std.Progress.Node)
             const out_pdb = self.pdb_out_path orelse try allocPrint(arena, "{s}.pdb", .{
                 full_out_path[0 .. full_out_path.len - out_ext.len],
             });
+            const out_pdb_basename = std.fs.path.basename(out_pdb);
```

Comment:
I don't think it should be a basename. It should be a path relative to the project directory, or a path relative to the global cache directory.

If an absolute path gets to this line of code, it's already too late. The Cache.Directory or Package.Path abstractions need to be used to retain the directory information to this point.

Reply:
So, store `o/hash/file.pdb` or `zig-out/bin/file.pdb`?

It might be possible<sup>*</sup>, but I'm not sure what the benefit of this is when using the basename is consistent. Even if another direction is wanted in the future I think this patch improves things, as currently `build-exe` will result in a binary that depends on cwd to get a proper stack trace.

<sub>* using the cache would make more sense in the compiler, but I'm not sure how the runtime stack trace logic would work<sub>